docs/howto/bootstrap/use.md

```diff
@@ -83,4 +83,154 @@ flowchart LR
     SSVC does not prescribe any particular response process, but it does provide a structured way to make decisions 
     within the response process.
 
-{% include-markdown '../communicating_results.md' heading-offset=1 %}
```

Comment:
As I was fixing links, I also took advantage of some opportunities to eliminate the need for some file includes by just merging the included content into the parent file.